projects/payment-lib/src/lib/services/xl-file/xl-file.service.ts

```diff
@@ -173,9 +173,10 @@ export class XlFileService {
     if (value) {
         // Remove tabs, carriage returns, and newlines
         value = value.replace(/^[\t\r\n@]+|[\t\r\n@]/g, (match, offset) => offset === 0 ? '' : ' ');
-        // Check if the first character is '=' and remove it
-        if (value.charAt(0) === '=') {
-          value = value.substring(1);
+
+        // Check if the first character is '-' or '=' or '+'
+        if (value.charAt(0) === '-' || value.charAt(0) === '=' || value.charAt(0) === '+') {
```

Comment:
Hi Olie, wouldn't something like the following be better, this is just an example. You can easily break your example by doubling up on the vulnerable characters.

const cell = "==Caclulator(1+1)";
const regexCell = /^['=\-+]+/i;
cell.replace(regexCell, '\'')